addLocation.html

```diff
@@ -138,11 +138,11 @@ <h3>Port</h3>
         function addLocation() {
             var domainName = document.getElementById("domainName").value;
             if (domainName == "") {
-                domainName = "Enter domain name";
+                domainName = "Enter place name";
             }
             var owner = document.getElementById("owner").value;
             if (owner == "") {
-                owner = "Enter owner of domain";
+                owner = "Enter owner of place";
```

Comment:
Seems grammatically awkward. Also, is the "enter" bit necessary?

There are more places with this, but I only marked this one.

Reply:
I do not know actually. I just replaced "domain" in all strings where it made sense. I have no idea how to actually get to all of these different UIs.

Reply:
This has simply be changed to "Location Name" in more recent builds from: https://github.com/vircadia/vircadia/issues/238